### ISSUE TITLE
Late visibility of changes 

### ISSUE THREAD

**Issue by Alexander-Dubrawski** [hyrise/Cockpit#755]
**Please describe the bug:**  
If we start a workload in the frontend the changes (throughput, latency,...) are visible just after around 20 to 25 seconds in the frontend. At the moment I am not sure if the problem lies in the frontend or backend (or even docker). 
I think we should further investigate the problem. 

**Please describe steps to reproduce:**  
1. start the docker setup
2. Add a database in the frontend
3. load the data (tpch sf 0.1)
4. start a workload 

**What is the expected behavior? Please describe:**  
The change in latency or throughput should be visible at least after 3 to 4 second (three seconds is the offset for the timestamp in the influx)

**Additional context:**  
I tested it with a VPN connection and the cockpit was running on a VM. Maybe this had an influence as well on the performance.

**Comment by Bouncner** [comment]
Didn't see that issue last time I tested the cockpit (around 1-2 weeks ago). I'll check it later today ...

**Comment by Alexander-Dubrawski** [comment]
I investigated the issue. I found out that the problem doesn't exist if we:

* run the cockpit in docker or native on the local machine. 
* If the Hyrise runs in docker, native, or on a VM. 

We only get this strange delay if we run the cockpit on a VM. The backend seems to be responsive at all time and if I ask for the throughput directly from the backend I get the current numbers without a delay, but in the frontend, they are shown with a delay. So I think the problem lies in the frontend, but I can't really tell where.
